dashboard/client/average_response_time/average_response_time.html

```diff
@@ -0,0 +1,6 @@
+<template name="averageResponseTime">
+  <h2>Average response time</h2>
+  <div id="average-response-time">
```

Comment:
The file contains **6 lines**. In this case, readability is good and I don't see any real reasons to add a redundant empty line.